### ISSUE TITLE
InputTypeObject not validate NonNull 

### ISSUE THREAD

**Issue by swumao** [Youshido/GraphQL#190]
I defined an InputType like : 
```php
class EventInputType extends AbstractInputObjectType
{
    public function build($config)
    {
        $config->addField('group_id', new NonNullType(new IntType()))
            ->addField('title', new NonNullType(new StringType()))
            ->addField('description', new NonNullType(new StringType()))
            ->addField('start_time', new NonNullType(new StringType()))
            ->addField('end_time', new NonNullType(new StringType()))
            ->addField('type', new NonNullType(new IntType()))
            ->addField('user_count_limit', new NonNullType(new IntType()))
            ->addField('address', new NonNullType(new StringType()));
    }
}
```
and in my CreateEventField class's build method:
```php
    public function build(FieldConfig $config)
    {
        $config->addArgument('event', new NonNullType(new EventInputType()));
    }
```
and even I post mutation like this the resolve method has been execute:
```
mutation {
	createEvent(event: 1)
}
```

Is that a bug?

**Comment by portey** [comment]
Yes, looks like a bug. I'll check it tonight.

**Comment by portey** [comment]
done, merged to v2.0 (wait for release)

**Comment by swumao** [comment]
thx
